posthog/management/commands/test_migrations_are_safe.py

```diff
@@ -15,6 +15,7 @@ def run_and_check_migration(variable):
                 sql = call_command("sqlmigrate", results[0], results[1])
                 if (
                     re.findall(r"(?<!DROP) (NOT NULL|DEFAULT)", sql, re.M & re.I)
+                    and not re.match(r"(?<!CREATE UNIQUE INDEX) (NOT NULL)", sql)
```

Comment:
Was having trouble with the regex of finding all "NOT NULL" instances that aren't followed by CREATE UNIQUE TOGETHER / ALTER TABLE so hoping to get this in and then take it out after the PR is merged 🤗